connect/clients/kafka.py

```diff
@@ -187,7 +187,8 @@ async def get_message_from_kafka_cb(self, callback_method) -> None:
             #     message = combine_segments(msg.value(), self._generate_header_dictionary(msg.headers()))
 
             if message is not None:
-                logger.info(
+                logger.log(
+                    TRACE,
```

Comment:
Instead of subclassing Logger to add a trace() method, we use logger.log(TRACE, ...)

Reply:
It's great that we don't have to sub-class! I was also looking at using a partial function or adding a "trace" function to the startup server handler, neither of which really panned out. if we find that having to do a "log" with a trace constant/config is too much of a pain we can always subclass down the line.